definitions/divorce/json/CaseField.json

```diff
@@ -3366,5 +3366,21 @@
     "Label": "### Respondent's Solicitor Details",
     "FieldType": "Label",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2019",
+    "CaseTypeID": "DIVORCE",
+    "ID": "labelPetGenderChangeWarning",
```

Comment:
do you need to update permissions for this field?

Reply:
Hmm you would think so actually - but it still seems to work. I'll add anyway.

Reply:
Fixed that @qzhou-hmcts